src/views/AccountDetails.vue

```diff
@@ -0,0 +1,409 @@
+<template>
+    <div class="card-content">
+      <div class="media">
+        <div class="media-content">
+          <div class="columns">
+            <div class="column">
+            </div>
+            <div class="column">
+              <div class="field is-grouped is-pulled-right">
+                <p class="control">
+                  <button class="button is-small save-document-button"
+                          title="Save changes"
+                          data-qa="save-changes"
+                          :disabled="parseInt(userDetails.acl) > 20">
+                    <span class="icon is-small">
+                      <i class="fas fa-save"></i>
+                    </span>
+                    <span> Save </span>
+                  </button>
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <hr/>
+      <div v-show="!loadingDocCounter && selectedBranch && accountId">
+        <div class="content collapsible-content px-5 py-5">
```

Comment:
This div isn't collapsible

```suggestion
        <div class="content">
```